tests/cases/ApiRoutesResolverTest.php

```diff
@@ -1,30 +1,32 @@
 <?php
 
-namespace Contributte\ApiDocu\Tests\Cases;
+declare(strict_types=1);
+
+namespace Tests\Cases;
 
-use Tester\TestCase;
-use Tester\Assert;
-use Mockery;
 use Contributte\ApiRouter\ApiRoute;
-use Nette\Application\Routers\RouteList;
-use Nette\Application\Routers\Route;
 use Contributte\ApiRouter\DI\ApiRoutesResolver;
 use Contributte\ApiRouter\Exception\ApiRouteWrongRouterException;
+use Nette\Application\Routers\Route;
+use Nette\Application\Routers\RouteList;
+use Tester\Assert;
+use Tester\TestCase;
+use Throwable;
```

Comment:
Could you please use native php classes directly in code prefixed with `\`? (`} catch (Throwable $e) {`)

Reply:
ok